File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;

    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height >= 9);

    for (int i = 1; i <= height; i++)
    {
        int spaces = height;

        while (spaces > i)
        {
            spaces--;
            printf(" ");
        }

        for (int left = 0; left < i; left++)
        {
            printf("#");
        }
        printf(" ");
        printf(" ");

        for (int right = 0; right < i; right++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: pset6/mario/more/mario.py
from cs50 import get_int


def main():
    """Asks the user for the height"""
    while True:
        height = get_int("Height: ")
        if (height > 0 and height < 9):
            break

    """Prints the pyramide"""
    for i in range(height):
        hashes = i + 1
        printSpace(height-1-i)
        printHash(hashes)
        printSpace(2)
        printHash(hashes)
        print()


def printSpace(num):
    """Print num spaces"""
    for i in range(num):
        print(" ", end="")
        
        
def printHash(num):
    """Print num hashes"""
    for i in range(num):
        print("#", end="")
        
        
if __name__ == "__main__":
    main()